deltachat-ios/Controller/ContactDetailViewController.swift

```diff
@@ -430,3 +440,20 @@ class ContactDetailViewController: UITableViewController {
         }
     }
 }
+
+extension ContactDetailViewController: MultilineLabelCellDelegate {
+    func phoneNumberTapped(number: String) {
+        let sanitizedNumber = number.filter("0123456789".contains)
```

Comment:
what about '+' ? doesn't ios handle +1 for example correctly?

Reply:
i just copy+pasted from the existing code :)